lib/open_food_network/feature_toggle.rb

```diff
@@ -10,6 +10,17 @@ module FeatureToggle
     # Please add your new feature here to appear in the Flipper UI.
     # We way move this to a YAML file when it becomes too awkward.
     # **WARNING:** Features not in this list will be removed.
+    #
+    # Once the feature is ready for general production use,
+    # copy the feature declaration to ACTIVE_BY_DEFAULT below and
+    # activate it for all instances with a migration:
+    #
+    #   ./bin/rails generate migration EnableFeatureDragonMode
+    #
+    # Replace the `change` method with an `up` method and add this line:
+    #
+    #   Flipper.enable("dragon_mode")
+    #
```

Comment:
I looked into writing our own generator for this type of migration but it took too long to figure out how to do it correctly. Editing the migration manually is really not a big deal, I think.